src/test/java/uk/gov/hmcts/cp/config/OpenAPIConfigurationLoaderTest.java

```
@@ -54,7 +53,7 @@ void loadOpenApiFromClasspath_should_throw_for_missing_resource() {
 
     @Test
     void loadOpenApiFromClasspath_should_throw_for_blank_path() {
```

I like use of  @DisplayName annotation to make the tests more readable. Your thoughts on adopting it? 
 @DisplayName("Missing resource should throw an exception when path is blank")

---

Maybe ... I like to just use the method name to do that 
My tests are generally more like 
shouldThrowExceptionWhenPathIsBlank

This is not my tests ... its existing tests by Duncan.
And looks like Duncan pref is 
xxxx_should_dosomething
Which actually I quite like .. the underscores help break it up better than caps case

So I have amended the 2 tests that I did add to be ...
generateOpenApiModels_should_create_Judges()
and
generateOpenApiModels_should_create_ErrorResponse()
( more like Duncans :) )